Code/IO/include/sitkImageSeriesReader.h

```diff
@@ -40,6 +40,17 @@ namespace itk {
      * Once the image series is read the meta-data is directly
      * accessible from the reader.
      *
+     * \note If the pixel type for the returned image is not specified
```

Comment:
technically the pixel type is implicitly specified as "sitkUnknown", to indicate that is should be deduced.